Character.h

```diff
@@ -53,22 +54,17 @@ struct Character
         std::cout << getName() << "'s attack damage level has been boosted to " << attackDamage << std::endl;
     }
 
-    void printStats()
-    {
-        std::cout << getName() << "'s stats: " << std::endl;
-        std::cout << getStats(); //make your getStats() use a function from the Utility.h
-        
-        std::cout << std::endl;
-        std::cout << std::endl;
-    }
+    void printStats();
+
 protected:
     std::vector<std::unique_ptr<Item>> defensiveItems;
     std::vector<std::unique_ptr<Item>> helpfulItems;
+    std::vector<std::unique_ptr<Item>> attackItems;
```

Comment:
the DragonSlayer gets **one** attack item.  no other characters get an attack item.
add it as a member variable to that class.
- [ ] revise